examples/tv-casting-app/darwin/TvCasting/TvCasting/MCDiscoveryExampleView.swift

```diff
@@ -25,25 +25,30 @@ extension MCCastingPlayer : Identifiable {
 
 struct MCDiscoveryExampleView: View {
     @StateObject var viewModel = MCDiscoveryExampleViewModel()
+    @State private var selectedCastingPlayer: MCCastingPlayer?
+    // Enable navigating to the MCConnectionExampleView with or without the use
+    // Commissioner-Generated Passcode (CGP) flag.
+    @State private var navigateWithUseCGP = false
+    @State private var navigateWithoutUseCGP = false
```

Comment:
Not sure I understood the use of these two @State vars?

Reply:
The @State variables navigateWithUseCGP and navigateWithoutUseCGP are used to control navigation to MCConnectionExampleView with different configurations based on the type of user interaction (tap or long press). They enable the view to distinguish between two different ways of connecting to a CastingPlayer. We can simplify the code by using a single variable and distinguishing between the two gestures within the navigation link's isActive binding.